### PR TITLE
Namerd support for telemeters (#740)

### LINKED ISSUE #740: namerd should support telemeters
for stats export

### PR DESCRIPTION
**Pull request by esbie**
This cl adds a "telemetry" section to the namerd config that behaves the same way
as linkerd's telemetry section. Fixes #740

### CODE REVIEW

File: namerd/docs/config.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Key | Required | Description
 [interfaces](#interfaces) | no | Configures namerd's published network interfaces.
 [storage](#storage) | yes | Configures namerd's storage backend.
 [namers](https://linkerd.io/config/head/linkerd#namers) | no | Configures namerd's integration with various service discovery backends. namerd uses the same namers as linkerd.
+[telemetry](https://linkerd.io/config/head/linkerd#telemetry) | no | Configures namerd's metrics instrumentation.
```
Comment (adleong) [inline namerd/docs/config.md]: Might be worth mentioning that tracing doesn't apply to namerd and any tracers provided by these telemeters will not be used.

Comment (esbie) [inline namerd/docs/config.md]: oh right good point

**Comment by esbie** [comment]
I tested a couple, but I can run through the rest

**Comment by esbie** [comment]
ok, given that half of our telemeters are tracing-only, I added a warning at startup.

**Pull request merged**